### PR TITLE
#1735 Add metrics data in the pipeline graph 

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1735)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineGraph/PipelineGraph.js [hunk 1]
```diff
@@ -14,147 +14,152 @@
...
-      const statusIcon = getStatusIcon(updateState);
-
-      const html = `<div class="node-graph ${topicCls} ${stateCls}">
-        <span class="node-icon"><i class="fa ${icon}"></i></span>
-        <div class="node-text-wrapper">
-          <span class="node-name">${name}</span>
-          <span class="node-status">Status: ${status}</span>
-          <span class="node-type">${displayKind}</span>
+const PipelineGraph = props => {
+  useEffect(() => {
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/PipelinePage/PipelineGraph/PipelineGraph.js]: Maybe can split to multiple useEffect ?
Look like to complex

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineGraph/PipelineGraph.js]: Good point :) I will try to separate this function to smaller chunks

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineGraph/PipelineGraph.js]: Simplified the `renderGraph` function a bit  :)

**Comment by konekoya** [comment]
retry manager-it

**Comment by chia7712** [comment]
retry

**Pull request merged**